engine/src/test/java/org/hibernate/search/test/configuration/mutablefactory/MutableFactoryTest.java

```diff
@@ -152,6 +155,77 @@ public void testAddingClassSimpleAPI() throws Exception {
 		sf.close();
 	}
 
+	@Test
+	public void testAddingClassSimpleAPIwithJMX() throws Exception {
+		SearchIntegrator sf = new SearchIntegratorBuilder().configuration( new SearchConfigurationForTest().addClass( A.class ).addProperty( "hibernate.search.jmx_enabled" , Boolean.TRUE.toString() ).addProperty( "hibernate.search.generate_statistics", Boolean.TRUE.toString() ).addProperty( "com.sun.management.jmxremote", Boolean.TRUE.toString() ) ).buildSearchIntegrator();
+
```

Comment:
Yes Sanne, you're right.
Just wrote it down at the beginning of the work and forgot to format in the right style.

Just pushed  the class well formatted.